makeflow/src/makeflow_log.c

```diff
@@ -275,7 +275,7 @@ void makeflow_log_recover(struct dag *d, const char *filename, int verbose_mode,
 	dag_count_states(d);
 
 	// Check for log consistency
-	if(!first_run) {
+	if(!first_run && !skip_file_check) {
```

Comment:
The only operation this loop does it to determine if a file should exist according to the log, and if it should we need to stat the file and make sure it does and wasn't modified. There is no added benefit to entering the loop if we aren't going to stat the file. Since AND is used in the comparison, the first set of checks say this should be here and then we verify with stat. If we don't verify, there is no reason to stat, let me know if this should off.